src/backport.ts

```diff
@@ -155,20 +155,24 @@ const getFailedBackportCommentBody = ({
   commitSha,
   errorMessage,
   head,
+  repo,
 }: {
   base: string;
   commitSha: string;
   errorMessage: string;
   head: string;
+  repo: string;
 }) => {
-  const worktreePath = `../.worktrees/backport-${base}`;
+  const worktreePath = `../.worktrees/${repo}/backport-${base}`;
```

Comment:
```suggestion
  const worktreePath = `\$(git rev-parse --show-toplevel)/.worktrees/${repo}/backport-${base}`;
```

(i'm not super fluent in typescript, but think that this should then include the actual command in the output?)